roles/cifmw_block_device/templates/ceph-osd-losetup.service.j2

```diff
@@ -9,7 +9,7 @@ After=syslog.target
 [Service]
 Type=oneshot
 ExecStart=/bin/bash -c '/sbin/losetup {{ cifmw_block_device_loop }} || \
-/sbin/losetup {{ cifmw_block_device_loop }} /var/lib/ceph-osd.img ; partprobe {{ cifmw_block_device_loop }}'
+/sbin/losetup {{ cifmw_block_device_loop }} {{ cifmw_block_device_image_file }} ; partprobe {{ cifmw_block_device_loop }}'
```

Comment:
Yes, we have the same default for this value and should use the variable in case someone overrides it.

https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/cifmw_block_device/defaults/main.yml#L21